src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsTransport.cs

```diff
@@ -80,7 +81,8 @@ public Task StartAsync(Uri url, Channel<byte[], SendMessage> application, Transf
             var response = await _httpClient.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cancellationToken);
 
             var stream = await response.Content.ReadAsStreamAsync();
-            var pipelineReader = stream.AsPipelineReader(cancellationToken);
+            var memoryPool = new MemoryPool();
```

Comment:
This should be shared.

Reply:
After looking at this a bit I think the best option is to have a static `MemoryPool` object in the SSE transport. The other way would be to pass it into the constructor which would require the `DefaultTransportFactory` to take in a `MemoryPool` or have a static one. And if it took one in then `HttpConnection` would also need to accept one and that seems like a lot of ugly API change to avoid a static.

Reply:
I think we should not block on this. We are trying to unblock the build - if we don't know the best way to deal with it now we should do the simplest thing that works now and find the right solution once we have the latest bits in dev.

Reply:
We aren't blocking anything yet, the gate still isn't running, this was just getting ahead and being prepared